src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1460,6 +1460,16 @@ This version used in: {2}</value>
   <data name="Implement_explicitly" xml:space="preserve">
     <value>Implement explicitly</value>
   </data>
+  <data name="cannot_be_null_or_empty" xml:space="preserve">
+    <value>cannot be null or empty</value>
```

Comment:
Please add a `{0}` place holder for replacement with the parameter name.  Constructing this string with concatenation may not make sense for all languages.